cafebabel/templates/articles/proposals/create.html

```diff
@@ -57,13 +57,13 @@ <h3>Wanna be part of Cafébabel project? That’s the right place</h3>
 
     <button type=submit>Suggest this article</button>
   </form>
-  <a class="button proposal-email" href=m&#x61;ilto:{{config.EDITORS_EMAIL_DEFAULT|urlencode|replace('.', '&#46;')|replace('a', '&#x61;')|safe}}>Or send your proposal by email</a>
+  <a class="button proposal-email" href=m&#x61;ilto:{{ EDITORS_EMAIL_DEFAULT|urlencode|replace('.', '&#46;')|replace('a', '&#x61;')|safe }}>Or send your proposal by email</a>
   <script>
-     const editorEmail = {{ editor_emails|to_json }}
+     const editorEmails = {{ EDITOR_EMAILS|to_json }}
```

Comment:
I was thinking that we may want to encrypt them here in their HTML version like at line 60

Reply:
Such a PITA but necessary, thanks 😘